tools/src/prepare-for-vale/KeepDescriptions.ts

```diff
@@ -62,4 +67,11 @@ export default class KeepDescriptions {
       return Array(match.length + 1).join(char)
     })
   }
+
+  remove_links(line: string): string {
+    return line.replace(/\[([^\]]+)\]\([^)]+\)/g, (match, p1) => {
+      const spaces = ' '.repeat(match.length - p1.length-1);
```

Comment:
```suggestion
      const spaces = ' '.repeat(match.length - p1.length - 1)
```

Suprised `npm run lint--fix` didn't pick on this. Remove `;` and space out `-`.